features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/main/java/org/opennms/features/topology/plugins/topo/linkd/internal/EnhancedLinkdTopologyProvider.java

```diff
@@ -53,7 +53,9 @@
 import org.opennms.features.topology.api.topo.AbstractSearchProvider;
 import org.opennms.features.topology.api.topo.AbstractVertex;
 import org.opennms.features.topology.api.topo.Criteria;
+import org.opennms.features.topology.api.topo.DefaultMetaInfo;
 import org.opennms.features.topology.api.topo.Edge;
+import org.opennms.features.topology.api.topo.MetaInfo;
```

Comment:
Do we need these imports? It is defined in AbstractTopologyProvider and not used in this class.